Base/home/anon/Source/js/date-strings.js

```diff
@@ -0,0 +1,17 @@
+let dateStrings = [
+    'Jan 01 1970 GMT',
+    'Wed Apr 17 23:08:53 2019 +0000',
+    '2021-07-01 03:00Z',
+    '2024-01-08 9:00Z',
+    'Wed, 17 Jan 2024 11:36:34 +0000',
+    'Sun Jan 21 2024 21:11:31 GMT 0100 (Central European Standard Time)',
+    '2024-01-15 00:00:01',
+    'Tue Nov 07 2023 10:05:55  UTC',
+    'Wed Apr 17 23:08:53 2019',
+    '2024-01-26T22:10:11.306+0000',
+    '1/27/2024, 9:28:30AM',
+];
+
+for (let dateString of dateStrings) {
+    console.log(new Date(dateString));
+}
```

Comment:
Please add this as a test under `Userland/Libraries/LibJS/Tests/` instead, so it actually gets run by our testers :)

Reply:
Specifically we test these ad-hoc extensions in: Userland/Libraries/LibJS/Tests/builtins/Date/Date.parse.js